WordPress/Classes/ViewRelated/NUX/SignupEpilogueCell.swift

```diff
@@ -115,9 +115,44 @@ class SignupEpilogueCell: UITableViewCell {
         cellField.backgroundColor = .basicBackground
     }
 
-    // MARK: - Private behavior
+}
+
+extension SignupEpilogueCell: UITextFieldDelegate {
+
+    func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
+        if let cellType = cellType, cellType == .displayName || cellType == .password {
+            let updatedText = NSString(string: textField.text!).replacingCharacters(in: range, with: string)
```

Comment:
Should textField.text be force unwrapped? I thought `shouldChangeCharactersIn range:` could be called after a user deletes all the text from the textField as well.

Reply:
> Should textField.text be force unwrapped?

Ah yes, that was from the original implementation. Tis been removed.

> I thought `shouldChangeCharactersIn range:` could be called after a user deletes all the text from the textField as well.

I'm not quite sure what this means. This method is called every time a character is added/deleted, so that would include when all characters are deleted. If I missed something let me know.

Also, while testing this change, I found a pre-existing bug (https://github.com/wordpress-mobile/WordPress-iOS/issues/14067), which I'll fix separately.